greenlet.js

```diff
@@ -59,8 +59,7 @@ export default function greenlet(asyncFunction) {
 			// The filter is to provide a list of transferables to send zero-copy
 			worker.postMessage([currentId, args], args.filter(x => (
 				(x instanceof ArrayBuffer) ||
-				(x instanceof MessagePort) ||
-				(x instanceof ImageBitmap)
```

Comment:
We could check for ImageBitmap support using (createImageBitmap && x instanceof ImageBitmap) I've used ImageBitmap in the past for WebGL textures so I think it is beneficial to keep supporting it.

Reply:
That's a much better solution. I can add this to the PR for sure. Thanks for taking the time to review